### PR TITLE
WRQ-10629: Removed deprecated APIs

### PR DESCRIPTION
**Pull request by 0x64**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
As major version upgrade, deprecated APIs are removed.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Removed deprecated APIs.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRQ-10629

### Comments
Enact-DCO-1.0-Signed-off-by: Seungcheon Baek (sc.baek@lge.com)

### CODE REVIEW

File: packages/webos/CHANGELOG.md [hunk 1]
```diff
@@ -23,6 +23,11 @@ No significant changes.
 ## [4.7.11] - 2024-05-13
 
 No significant changes.
+## [unreleased]
```
Comment (juwonjeong) [inline packages/webos/CHANGELOG.md:26]: Could you please check the CHANGELOG.md? I think it inserted in the wrong place.

File: packages/ui/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,19 @@
...
+### Removed
+
+- `ui/GridListImageItem`
+- `ui/IconButton`
+- `ui/Marquee.MarqueeDecorator` config `className`
+- `ui/RadioDecorator`
+- `ui/Scroller.ScrollerBase`
+- `ui/SlotItem`
+- `ui/ToggleItem`
+- `ui/VirtualList.UiVirtualListBase`
```
Comment (juwonjeong) [inline packages/ui/CHANGELOG.md:16]:
```suggestion
```
As https://github.com/enactjs/enact/pull/3063, I think we don't need to mention ui/VirtualList.UiVirtualListBase in changelog

**Comment by juwonjeong** [comment]
Please resolve conflicts in CHANGELOG.md

**Comment by juwonjeong** [comment]
I think we should remove some lines in codecov.yml 
- "packages/ui/Scrollable"
- "packages/ui/Scroller/UiScrollerBase.js"
- "packages/ui/GridListImageItem"
- "packages/ui/VirtualList/VirtualListBase.js"
- "packages/ui/VirtualList/UiVirtualListBase.js"

File: packages/core/platform/platform.js [hunk 3]
```diff
@@ -231,16 +84,10 @@ const parseUserAgent = (userAgent) => {
  * @property {Number} chrome - The version of the detected browser, if chrome browser is detected
  * @property {Object} [extra] - Additional information about the detected platform. Deprecated: will be removed in 5.0.0.
```
Comment (juwonjeong) [inline packages/core/platform/platform.js:85]: I think this line should be removed

**Comment by 0x64** [comment]
> I think we should remove some lines in codecov.yml
> 
> * "packages/ui/Scrollable"
> * "packages/ui/Scroller/UiScrollerBase.js"
> * "packages/ui/GridListImageItem"
> * "packages/ui/VirtualList/VirtualListBase.js"
> * "packages/ui/VirtualList/UiVirtualListBase.js"

Good catch! Thank you!

File: packages/ui/CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,19 @@
...
+- `ui/VirtualList.VirtualListBase`
```
[the remark this answers is not in the document]
Comment (0x64) [reply packages/ui/CHANGELOG.md:17]: 🫡

File: packages/ui/CHANGELOG.md (hunk 2 above)
Comment (0x64) [reply packages/ui/CHANGELOG.md:16]: 🫡

File: packages/webos/CHANGELOG.md (hunk 1 above)
Comment (0x64) [reply packages/webos/CHANGELOG.md:26]: Well, git cannot merge as humans expect. 😢 Thanks for pointing out.

File: packages/core/platform/platform.js (hunk 3 above)
Comment (0x64) [reply packages/core/platform/platform.js]: Sure!

**Review by juwonjeong — APPROVED** [review]
LGTM

**Comment by MikyungKim** [comment]
Sorry for the late review. Please resolve the merge conflicts.

**Review by MikyungKim — APPROVED** [review]
LGTM

File: packages/core/platform/platform.js [hunk 5]
```diff
@@ -220,27 +74,19 @@ const parseUserAgent = (userAgent) => {
 		deprecate({name: `supporting ${detectedInfo.browserName} version before ${supportedVersions[detectedInfo.browserName]}`, until: '5.0.0'});
```
Comment (seunghoh) [inline packages/core/platform/platform.js:74]: @0x64 Why the version is already '5.0.0'? `until` is also unclear for me.

**Review by seunghoh — COMMENTED** [review]
webos/platform/platform.js line 26~27 are not needed as legacy version
```
const platforms = [
	// LG webOS before adapting Chrome
	{regex: /Web0S;.*Safari\/537.41/, version: 1}, // using WebKit 537.41 for WebAppManager, using Chrome 26 for the browser app
	{regex: /Web0S;.*Safari\/538.2/, version: 2},  // using WebKit 538.2  for WebAppManager, using Chrome 34 for the browser app
```

File: packages/core/platform/platform.js (hunk 5 above)
Comment (0x64) [reply packages/core/platform/platform.js:74]: `until` makes a message like " Will be removed in ${until}" and the original message was correct. But, it looks not good to notify 'deprecate' along with versions as we will update the base browser versions annually.
For now, it looks better simply removing 'until' option.

**Comment by 0x64** [comment]
I removed UA for unsupported legacy versions.

**Pull request merged**